### ISSUE TITLE
audio sent via bluetooth when first opening app crashing BT device

### ISSUE THREAD

**Issue by simpsonar77** [daneren2005/Subsonic#811]
This is admittedly an odd issue, and only occurs on one of my bluetooth car receivers with my android (LG V10) device.

I have a device called a GROM Audio USB3 car bluetooth adapter.  This device plays both bluetooth and USB audio files.  USB based audio plays automatically if a USB device is connected.  When I open any media player other than dsub, everything is fine and the USB audio keeps playing until I initiate playback in the media app.   There are no interruptions in USB playback until then.

When I open DSUB, USB playback is cut off as though music is being played through DSUB via bluetooth, but then immediately stops (1-2 seconds later) and the device returns to playing USB audio.  Now when I press play on DSUB or any other media player app, nothing plays through the BT device.  However, the phone is still "sending" audio through the bluetooth connection as though everything is fine.  My only recourse is to restart the BT device (disconnecting/turning off BT on the phone doesn't work).

Basically, DSUB is doing something at open that is hanging the BT device.  This doesn't happen with any other car based BT interface.  I have heard the immediately BT "interruption" when opening DSUB with other devices, but they seem to handle it better.

This doesn't happen with the subsonic app.  Is there maybe a setting causing the transmission at app open?

It's a very weird and isolated issue that makes it so I can't use dsub (which I love) in this particular car.

**Comment by simpsonar77** [comment]
additional testing today showed that even if the dsub app is already open prior to connecting to the BT device, the issue will occur.
